<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>People_Email_Send</class>
 <widget class="QDialog" name="People_Email_Send">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send Emails</string>
  </property>
  <widget class="QPushButton" name="SendEmail_CancelButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_TitleText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Send Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_HeaderText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject</string>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_BodyText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Body</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SendEmail_HeaderBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Three New Albums</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendEmail_SendButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="SendEmail_BodyBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;Hey,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;Three new albums are available for purchase through our online store.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;They are:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;Number - Title&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;Number - Title&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;Number - Title&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;Enjoy!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;The Monstercat Team&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Latest albums:</string>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_AlbumNumber1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_AlbumNumber2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_AlbumNumber3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_AlbumTitle1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_AlbumTitle2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SendEmail_AlbumTitle3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SendEmail_RefreshButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SendEmail_CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>People_Email_Send</receiver>
   <slot>BackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SendEmail_SendButton</sender>
   <signal>clicked()</signal>
   <receiver>People_Email_Send</receiver>
   <slot>SendEmailButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SendEmail_RefreshButton</sender>
   <signal>clicked()</signal>
   <receiver>People_Email_Send</receiver>
   <slot>RefreshAlbums()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackButton()</slot>
  <slot>SendEmailButton()</slot>
  <slot>RefreshAlbums()</slot>
 </slots>
</ui>
